css/build.css

```
@@ -8,15 +8,19 @@ fl-record-container .record-container-test-mode {
-  color: #6A6868!important;
+  border: none !important;
+  color: #6a6868 !important;
   font-family: "Open Sans", sans-serif;
   font-size: 12px;
   padding: 15px;
   text-align: center;
-  padding: 0!important;
-  display: block!important;
-}
\ No newline at end of file
+  padding: 0 !important;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

padding is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)